web/index.html

```diff
@@ -265,6 +270,11 @@ <h2 class="f4">What does it mean if I get an error?</h2>
         } else {
             outText += "❌ Could not find the multihash in the dht\n"
         }
+        if (respObj.ProviderRecordFromPeerInIPNI === true) {
+            outText += "✅ Found multihash advertised in IPNI\n"
+        } else {
+            outText += "❌ Could not find the multihash in IPNI\n"
+        }

```

Comment:
I think it makes sense that people should see all green if they've done their job. This makes it look like you should be advertised in both systems.

Seems like it should either be:
1. Your data was advertised in Amino and your multiaddr was there
2. Your data was advertised in IPNI and your multiaddr was there

and either should give you a check, and we can have the text change to say whether it was IPNI or the DHT that the multiaddr was found in (and ignore the case of both since right now that's not a thing anyone does)

Reply:
Done